### DISCUSSION TITLE
keyball61で左手側8番のキーが反応しない

### DISCUSSION THREAD

**Discussion by mushuper** [Yowkees/keyball#293] [Q&A] [answered]
お世話になります。
先日keyball61を入手でき、初めての自作キーボードにチャレンジしました。
初心者ということもあり、初歩的な質問かもしれませんがご教示いただけますと幸いです。

ビルドガイドを参考に、一通りの組み立てが完了したのですが、左手側8番のキーが反応しない状態です（当方右側にトラックボールを設置しました）。
近接のダイオードなどは再度はんだ付けをしてみたのですが解消せずでした。

他に確認すべき箇所がありましたらご教示いただけますと幸いです。

![65A4B354-D13B-48FE-AC21-F9662D4B82B5](https://github.com/Yowkees/keyball/assets/140997560/7634e8b9-f26e-473b-ac94-b41894674d2f)

**Comment by Yowkees** [comment, marked as answer]
ご連絡ありがとうございます。
該当のキーソケットの半田付けはどうでしょうか？
キーソケットはとてもテンプラしやすく、きちんと端子と基板の間に半田を流し込む必要があります。
端子と基板の隙間を横からのぞきながら、半田も端子側面から溶かしてあげるとうまく流れ込みます。

あとはスイッチの足折れもご確認ください。

以上お手数ですがご対応お願いいたします。

**Reply by mushuper** [reply]
ご助言ありがとうございます。
仰っているとおり、端子と基盤の隙間にハンダを流し込めていませんでした。
一度ハンダを吸い取って再度つけ直したら、無事動作確認ができました。

この度はご対応いただきありがとうございました。
